README.md

```diff
@@ -23,8 +23,14 @@ $ git flow release publish 0.1.0
 $ git flow release finish 0.1.0
 ```
 
-To kick off the deployment, you'll still need to push the local tags remotely
-`git push --tags`
+After you've completed the `git flow` steps, you'll need to push the changes from your local `master` and `develop` branches back to the main repository.
+
+```bash
+$ git push origin develop
+$ git push origin master
```

Comment:
Makes sense. I just pushed another fixup which restores the `git checkout` lines.